### PR TITLE
Fix PackageRef dependencies

### PR DESCRIPTION
**Pull request by chenriksson**
Fixes: https://github.com/NuGet/NuGetGallery/issues/6348
Depends on: https://github.com/NuGet/ServerCommon/pull/189

Before regression (https://github.com/NuGet/NuGetGallery/pull/6007):
https://dotnet.myget.org/feed/nuget-build/package/nuget/NuGetGallery.Core/4.4.5-master-32624

After regression (https://github.com/NuGet/NuGetGallery/pull/6007):
https://dotnet.myget.org/feed/nuget-build/package/nuget/NuGetGallery.Core/4.4.5-master-33375

This PR:
https://dotnet.myget.org/feed/nuget-build/package/nuget/NuGetGallery.Core/4.4.5-chenriks-deps-39343

Note that some agents may need update before I merge... so far I've seen 1 work, and 1 fail.

### CODE REVIEW

File: src/NuGetGallery.Core/NuGetGallery.Core.csproj [hunk 1]
```diff
@@ -233,25 +235,23 @@
...
     </PackageReference>
-    <PackageReference Include="NuGet.Frameworks">
-      <Version>4.8.0-preview4.5287</Version>
+    <PackageReference Include="Microsoft.WindowsAzure.ConfigurationManager">
+      <Version>3.1.0</Version>
     </PackageReference>
-    <PackageReference Include="NuGet.Packaging">
-      <Version>4.8.0-preview4.5287</Version>
+    <PackageReference Include="NuGet.Build.Tasks.Pack">
+      <Version>5.0.0-preview1.5429</Version>
```
Comment (joelverhagen) [inline src/NuGetGallery.Core/NuGetGallery.Core.csproj]: Can we have a stable 4.x?

File: src/NuGetGallery.Core/NuGetGallery.Core.csproj [hunk 2]
```diff
@@ -233,25 +235,23 @@
...
-    </PackageReference>
     <PackageReference Include="Microsoft.Data.Services.Client">
       <Version>5.7.0</Version>
     </PackageReference>
     <PackageReference Include="Newtonsoft.Json">
       <Version>9.0.1</Version>
     </PackageReference>
-    <PackageReference Include="NuGet.Common">
-      <Version>4.8.0-preview4.5287</Version>
+    <PackageReference Include="Microsoft.Web.Xdt">
```
Comment (joelverhagen) [inline src/NuGetGallery.Core/NuGetGallery.Core.csproj]: I would try removing this one. This was a NuGet.Core 2.x dependency.

File: build.ps1 [hunk 3]
```diff
@@ -10,7 +10,7 @@ param (
     [string]$PackageSuffix,
     [string]$Branch,
     [string]$CommitSHA,
-    [string]$BuildBranch = '9f570166c49e93a911e54505a8041f3a123e69b1'
+    [string]$BuildBranch = 'chenriks-pack'
```
Comment (loic-sharma) [inline build.ps1]: >chenriks-pack [](start = 28, length = 13)

Don't forget to fix this guy

**Comment by chenriksson** [comment]
@joelverhagen @loic-sharma Thanks for reviewing!

I had to update/modify the VS installation on 3/5 agent VMs to allow this... but it now builds successfully on all agents. Tested using the signed MyGet definition.

**Pull request merged**